message_adapter/message_adapter.py

```diff
@@ -277,29 +286,25 @@ def __resolveConfigObject(self, event, config):
         * @param {*} config A config object, containing paths
         * @returns {*} A config object with all JSONPaths resolved
         """
-        try:
-            unicode
-        except NameError:
-            if isinstance(config, str):
-                return self.__resolvePathStr(event, config)
-        else:
-            if isinstance(config, unicode):
```

Comment:
Since we are getting rid of this conditional, is there any state where `config` could be a `unicode` in Py3?

Reply:
You've clarified that byte strings and text strings are one and the same in Py3. Thanks!